### PR TITLE
update installation instructions

### PR DESCRIPTION
**Pull request by alanb2718**
One question: the directory `installation` includes a starter `auto-config.inc.php` file, with a different name (`initial-auto-config.txt`). The user needs to upload this, change the name to `auto-config.inc.php`, and edit as needed. Is this the right way to do this, or would it be better to call it `auto-config.inc.php` right off in the directory `installation` and avoid needing to rename it?

### CODE REVIEW

File: installation/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Setting Up an Empty MySQL Database
+
+Set up an empty MySQL database, along with a MySQL user that has access to it.  (The directions for this step in the older tutorial are still valid.) The standard name for this database is `rootserver`, but you can name it something else if you prefer.
+
+## Uploading the BMLT Server Zip File
+
+Get the current version of the server from https://bmlt.app/downloads/, and upload it to your web hosting provider's server. (The directions for this step in the older tutorial are also still valid.) For this part of the step, upload the zip file *without* unzipping it on your local machine. Then unzip it on your server. You should end up with a directory `main_server` under the directory that holds the files that show up on your website. Thus, if your web hosting server has a directory `public_html` for the files that show up on your website, put `main_server` in that directory, like this: `public_html/main_server`.
```
Comment (pjaudiomv) [inline installation/README.md:14]: I would prob just link to https://github.com/bmlt-enabled/bmlt-root-server/releases for latest version

File: installation/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This step is different from the old tutorial.
+
+In the unzipped version of the BMLT Server on your local machine, locate the directory `installer` and find the file `initial-database.sql` in that directory.  Import the contents of this file into the empty MySQL database that you set up in the first step.  (If you are using cPanel, find the `phpMyAdmin` tool under `Databases`, select your new database, and then click `Import`.)
+
+## Adding the auto-config File
+
+This step is also different from the old tutorial.
+
+In the unzipped version of the BMLT Server, look again in the directory `installer` and find the file `initial-auto-config.txt`. Upload this file to your server, put it in the directory that holds your `main_server` directory, and rename it to `auto-config.inc.php`.  This file should have the permissions `-rw-r--r--` (`0644` in octal). This means that the owner of the file can read and write it, and the owning group and others can read it.
```
Comment (pjaudiomv) [inline installation/README.md:28]: I would just name it auto-config.inc.php by default and set the permissions on it to 0644 as well

**Review by pjaudiomv — APPROVED** [review]
added couple nit suggestions, take em or leave em. Thank you this looks really good

File: installation/README.md (hunk 1 above)
Comment (alanb2718) [reply installation/README.md]: OK done

**Pull request merged**